tests/integration/src/test/resources/org/mule/test/integration/routing/inbound/idempotent-router-with-filter-flow.xml

```diff
@@ -11,7 +11,7 @@
         <message-filter>
             <payload-type-filter expectedType="java.lang.String"/>
         </message-filter>
-        <idempotent-message-filter/>
+        <idempotent-message-validator/>
```

Comment:
I don't see any test cases where DUPLICATED_MESSAGE (FILTERED) error type is being thrown, and then handled in an error-handler.  Isn't this important given it's the use case where this would typically be being used?